tests/framework/testutils/log_observer.go

```diff
@@ -46,8 +46,8 @@ func NewLogObserver(level zapcore.LevelEnabler) (zapcore.Core, *LogObserver) {
 	}
 }
 
-// Expect returns the first N lines containing the given string.
-func (logOb *LogObserver) Expect(ctx context.Context, s string, count int) ([]string, error) {
+// ExpectAtLeastNTimes returns the first N lines containing the given string.
+func (logOb *LogObserver) ExpectAtLeastNTimes(ctx context.Context, s string, count int) ([]string, error) {
```

Comment:
Please update the Expect function instead. 
Current implementation of expect should be renamed to `ExpectNMatches`